components/prism-lawtex.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.lawtex = {
+  'comment': [
+    {
+      pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+      lookbehind: true,
+      greedy: true
+    },
+    {
+      pattern: /(^|[^\\:])\/\/.*/,
+      lookbehind: true,
+      greedy: true
+    }
+  ],
+  'string': {
+    pattern: /(^|[^\\](?:\\\\)*)(")(?:\\[\s\S]|\$\([^)]+\)|\$(?!\()|`[^`]+`|(?!\2)[^\\`$])*\2/,
+    lookbehind: true,
+    greedy: true,
+    inside: {
+      'escape': /\\./
+    }
+  },
+  'keyword': /\b(?:metainfo|head|body|foot|extra|where|extends|externs|template|list|struct|dependency|branch|topic|loop|period|if|else|elseif|foreach|print|use)\b/,
+  'important': /\b(?:this|value|id|mask|transient|itemsOnPage|language|style|type|default|description|tags|statement|overview|components|audience|inputs|functionalities|warnings|upgrades|title|name|label|order|mandatory|version|index|lower|upper|separator|request|element|help|tip|atomic|fields|loaders|options|key|operation|declarations|operations)\b/,
+  'logical': /\b(?:NOT|AND|OR|XOR|EXISTS|FORALL|IN)/,
+  'class-name': /\b(?:List|Object|String|Integer|Real|Boolean|Date|Time|Text|Document|Currency|Vector|Struct)\b/,
+  'function': /\b[A-Z].*?\b/,    
+  'boolean': /\b(?:true|false|null)\b/,
+  'operator': /(?:\[|\]|==|~=|<=|>=|<|>|\|)/,
+  'assignment': /=/,
+  'arithmetic': /(?:\-|\+|\&|\*|\/|%|\^)/,
```

Comment:
Is the distinction really necessary or could "operator" "assignment", "arithmetic", and "logical" be combined into one token? Is this useful for highlighting?

Reply:
No it is not! I simply copied the highlighter currently used. For some reason these three are declared separately but attributed to the same color.  Thank you.